server/gokbg3/src/main/groovy/org/gokb/UpdatePkgTippsRun.groovy

```diff
@@ -705,50 +836,40 @@ class UpdatePkgTippsRun {
            tid    : tippJson.titleId,
            tStatus: status_current]
       )
-      tipps = tippList
-      if (tipps.size() > 0) {
+      if (tippList.size() > 0) {
         log.debug("found by titleId in DB")
-        return tipps
-      }
-    }
-    // remap JSON Identifiers to [type: value]
-    def jsonIdMap = [:]
-    tippJson.identifiers.each { jsonId ->
-      jsonIdMap[jsonId.type] = jsonId.value
-    }
-    if (jsonIdMap.size() == 0) {
-      tippJson.title.identifiers.each { jsonId ->
-        jsonIdMap[jsonId.type] = jsonId.value
+        return tippList
       }
     }
+
     // search for package provider namespace identifier
     IdentifierNamespace providerNamespace = Package.get(pkg.id).provider?.titleNamespace
     if (providerNamespace && jsonIdMap[providerNamespace.value]) {
       // elastic search
-      TypeConvertingMap map = [
-          componentType     : 'TitleInstancePackagePlatform',
-          identfiers        : providerNamespace.value + ',' + jsonIdMap[providerNamespace.value],
-          pkg               : pkg.uuid,
-          platform          : tippJson.hostPlatform.uuid,
-          skipDomainMapping : true
-      ]
-
-      def something = esSearchService.find(map)
-
-      if (something.records?.size() > 0) {
-        log.debug("found by provider namespace ID in ES")
-        something.records.each {
-          def tipp = TitleInstancePackagePlatform.findByUuid(it.uuid)
-
-          if (tipp) {
-            tipps << tipp
-          }
-          else  {
-            log.warn("ES record TIPP ${it.uuid} does not exist!")
-          }
-        }
-        return tipps
-      }
+      // TypeConvertingMap map = [
+      //     componentType     : 'TitleInstancePackagePlatform',
+      //     identfiers        : providerNamespace.value + ',' + jsonIdMap[providerNamespace.value],
+      //     pkg               : pkg.uuid,
+      //     platform          : tippJson.hostPlatform.uuid,
+      //     skipDomainMapping : true
+      // ]
+
+      // def something = esSearchService.find(map)
+
+      // if (something.records?.size() > 0) {
+      //   log.debug("found by provider namespace ID in ES")
+      //   something.records.each {
+      //     def tipp = TitleInstancePackagePlatform.findByUuid(it.uuid)
+
+      //     if (tipp) {
+      //       tipps << tipp
+      //     }
+      //     else  {
+      //       log.warn("ES record TIPP ${it.uuid} does not exist!")
+      //     }
+      //   }
+      //   return tipps
+      // }
```

Comment:
Maybe we can add a comment on how long we are wanting to keep out-commented code in the future. Sometimes, these blocks get really old and every time you pass by you ask yourself: Should I delete that?

Reply:
Great Idea, I added comments with version info.